syntax = "proto3";
package ammb;

message MeshcoreMessage {
  string destination_meshtastic_id = 1;
  string payload = 2;
  int32 channel_index = 3;
  bool want_ack = 4;
}
